### PR TITLE
UT(iOS): Added UT's for snap shotter class

### PR DESCRIPTION
**Pull request by anil-shukla-axway**
https://jira.appcelerator.org/browse/MOD-2620

### CODE REVIEW

**Comment by m1ga** [comment]
nice to see all those tests :+1: makes it more reliable.

Hopefully the other features will get merged some time too :smile: A lot of good things are waiting and I don't want to make custom builds all the time ;-) !

File: test/unit/specs/snapshotter.spec.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+			longitude: -123.12494706,
+			latitudeDelta: 0.4,
+			longitudeDelta: 0.4,
+		},
+		size: { height: 200, width: 300 },
+		showsBuildings: true,
+		showsPointsOfInterest: false,
+	});
+
+	describe('ti.map.Snapshotter', function () {
```
Comment (fasihul-axway) [inline test/unit/specs/snapshotter.spec.js:17]: @anil-shukla-axway, its better to add a describe tag which can tell wether UT is for methods, property or constraint. Something like

`describe('methods', () => {
}`

File: test/unit/specs/snapshotter.spec.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+		});
+
+		it('should have function setShowsPointsOfInterest', () => {
+			expect(snapshotter.setShowsPointsOfInterest).toEqual(
+				jasmine.any(Function)
+			);
+		});
+
+		it('should have function takeSnapshot', () => {
+			expect(snapshotter.takeSnapshot).toEqual(jasmine.any(Function));
```
Comment (fasihul-axway) [inline test/unit/specs/snapshotter.spec.js:45]: Can you write a test case to see, if it is taking snapshot or not? One with invalid values if possible.

File: test/unit/specs/snapshotter.spec.js (hunk 1 above)
Comment (anil-shukla-axway) [reply test/unit/specs/snapshotter.spec.js]: Done

File: test/unit/specs/snapshotter.spec.js (hunk 2 above)
Comment (anil-shukla-axway) [reply test/unit/specs/snapshotter.spec.js]: Added UT please verify.

**Pull request merged**